### PR TITLE
Make phones functional

### PR DESCRIPTION
**Pull request by Chief-Engineer**
## About the PR 

Modifies prayer system to allow it to be used for other similar functions. Adds this functionality to the red phone and to the banana phone.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, or this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: The red phone can now be used to leave a message for Central Command
- tweak: The banana phone can now be used to leave a message for the Honkmother

### CODE REVIEW

File: Content.Server/Prayer/PrayableComponent.cs [hunk 1]
```diff
@@ -12,6 +12,34 @@ public sealed class PrayableComponent : Component
         [DataField("bibleUserOnly")]
         [ViewVariables(VVAccess.ReadWrite)]
         public bool BibleUserOnly;
+
+        /// <summary>
+        /// Message given to user to notify them a message was sent
+        /// </summary>
+        [DataField("sentMessage")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        public string SentMessage = "Your message has been sent to the gods...";
```
Comment (moonheart08) [inline Content.Server/Prayer/PrayableComponent.cs:21]: Localization.

File: Content.Server/Prayer/PrayableComponent.cs [hunk 2]
```diff
@@ -12,6 +12,34 @@ public sealed class PrayableComponent : Component
...
+
+        /// <summary>
+        /// Prefix used in the notification to admins
+        /// </summary>
+        [DataField("notifiactionPrefix")]
+        [ViewVariables(VVAccess.ReadWrite)]
+        public string NotifiactionPrefix = "PRAYER";
```
Comment (moonheart08) [inline Content.Server/Prayer/PrayableComponent.cs:28]: Localization.

File: Content.Server/Prayer/PrayableComponent.cs [hunk 3]
```diff
@@ -12,6 +12,34 @@ public sealed class PrayableComponent : Component
...
+
+        /// <summary>
+        /// Used in window title and context menu
+        /// </summary>
+        [DataField("verb")]
+        [ViewVariables(VVAccess.ReadOnly)]
+        public string Verb = "Pray";
```
Comment (moonheart08) [inline Content.Server/Prayer/PrayableComponent.cs:35]: Localization.

**Review by moonheart08 — CHANGES_REQUESTED** [review]

**Comment by ZoldorfTheWizard** [comment]
@Chief-Engineer is this still a valid pr with #13321 close to completion?

**Review by ZoldorfTheWizard — APPROVED** [review]

**Review by ZoldorfTheWizard — APPROVED** [review]

**Pull request merged**